tests/golden/xfstests

```diff
@@ -9,9 +9,9 @@ generic/011
 generic/013
 generic/014
 generic/020
-generic/023
-generic/024
 generic/028
+generic/029
+generic/030
```

Comment:
I guess this would be coordinated with updating our xfstests?

Reply:
Hmm I'll double check all these to make sure this isn't from an e9 branch....

Reply:
I swuashed the patch that re-enables mmap() xfstests and the one that adds the new `golden` output and it all properly corresponds now. There's no new branch needed here - this is all el8/el7 tests from the `scoutfs` branch.